quantum/quantum.c

```diff
@@ -957,6 +957,9 @@ void matrix_init_quantum() {
   #ifdef RGB_MATRIX_ENABLE
     rgb_matrix_init();
   #endif
+  #ifdef MODULE_ADAFRUIT_BLE
```

Comment:
Should this be defaulted to auto? 

I think it would be better to hide this behind a define, to configure it's status